### PR TITLE
M1 review

### PR DESCRIPTION
**Pull request by hfboyce**
Alterations for consistency and question amendments.

### CODE REVIEW

**Comment by ttimbers** [comment]
On module 1, chapter 2, the first question's formatting appears off (at least using Netlify's preview rendering):

Screen Shot 2021-01-19 at 11 05 05 AM

**Comment by hfboyce** [comment]
@ttimbers I fixed the above. It's just rendering.

**Comment by ttimbers** [comment]
This is also weird - the text code output (black box):

Screen Shot 2021-01-19 at 11 18 28 AM

**Comment by ttimbers** [comment]
Can we get them to make a simple plot in module 1, chapter 4? That would be very rewarding and motivating I think. Or is this a technical limitation?

**Comment by ttimbers** [comment]
Not sure if I agree with this question in module 1, chapter 7:

#### Question 1
When combining plots in Altair, we say that each plot has its own ___.
- Overlay
- Layer
- Panel

To me a plot == Chart, and in altair, a chart is a layer. And we layer compatible charts on top of each each other to build more complex plots.  So a plot having a layer sounds a bit odd to me... So I think we need to modify or replace this question?

**Comment by ttimbers** [comment]
Can we bold "Before going further take a look at the stock data." in module 1, exercise 9 to make it extra clear this needs to be done. Maybe even add, click Run code to do this before uncommenting the plot code and creating the visualization.

**Comment by ttimbers** [comment]
Also, for module 1, question 9, when I click submit and run the code to see if I got it right, the plot disappears and then I cannot easily answer the next question:

Screen Shot 2021-01-19 at 12 17 31 PM

**Comment by ttimbers** [comment]
This also happens in module 1, question 10.

**Comment by ttimbers** [comment]
As a learner, I do not like the "Use the code from exercise 9 above as a base." (I saw this in module 1, question 10). because when you go back, your answer you typed out is gone, and so you have to redo it, then remember to copy it. Let's avoid these and just give them the scaffolding needed.

**Review by ttimbers — CHANGES_REQUESTED** [review]
Some more smallish things to address.

File: chapters/en/slides/module1/module1_01.Rmd [hunk 1]
```diff
@@ -69,30 +70,36 @@ for set_name, df in anscombe.groupby('Set'):
...
 representations of the same data.
 
 For example, have a look at the
 four sets of numbers in the table on the slide.
+
 This is a slightly modified version of the original,
-[which was put together by statistician Francis Anscombe in the
-70s.](https://en.wikipedia.org/wiki/Anscombe%27s_quartet)
+<a href="https://en.wikipedia.org/wiki/Anscombe%27s_quartet" target="_blank">which was put together by statistician Francis Anscombe in the
+70s.</a>
```
[the remark this answers is not in the document]
Comment (joelostblom) [reply chapters/en/slides/module1/module1_01.Rmd]: Ok let's add parentheses around it

File: chapters/en/slides/module1/module1_03.Rmd [hunk 2]
```diff
@@ -1,333 +1,566 @@
...
-To combine two different types of graphical marks
-(line and point in this case),
-we will use Altair's *layering* grammar.
-In this slide,
-we start by defining each chart separately:
-first a line plot,
-then a point plot.
-We can then use `+` (plus) operator
-to combine the two into a layered chart.
+To enrich this display of information further,
```
[the remark this answers is not in the document]
Comment (joelostblom) [reply chapters/en/slides/module1/module1_03.Rmd:469]: Yes!

File: chapters/en/slides/module1/module1_03.Rmd [hunk 3]
```diff
@@ -1,333 +1,566 @@
...
+we can color the points according to a column in the dataframe.
+When we encode a column as the color channel
+Altair will automatically figure out an appropriate colorscale to use,
+depending on whether the data is numerical, categorical, etc.
+Here we have indicated that we want to color the points according to the car's horsepower,
+which indicated how powerful its engine is.
+
+We can see that the heavier cars have more powerful engines,
+than the lighter ones,
```
[the remark this answers is not in the document]
Comment (joelostblom) [reply chapters/en/slides/module1/module1_03.Rmd:478]: Yup let's comment on that too

**Comment by joelostblom** [comment]
> To me a plot == Chart, and in altair, a chart is a layer. And we layer compatible charts on top of each each other to build more complex plots. So a plot having a layer sounds a bit odd to me... So I think we need to modify or replace this question?

I think this does not matter much, so happy either way. According to the docs, we are technically layering *views of charts*. This makes it unclear if a chart *is*, *has*, or *exists in* a layer. In ggplot I believe a plot and a layer are the same. @hfboyce Let's change the wording to :

> Which function do we need to use to superimpose plots in Altair?
>
>- `overlay`
>- `layer`
>- `panel`

**Comment by hfboyce** [comment]
> Also, for module 1, question 9, when I click submit and run the code to see if I got it right, the plot disappears and then I cannot easily answer the next question. 

Can't find an easy solution to this since right now all the fixes I've try only returns the text `alt.Chart(...)` I'll talk to @joelostblom  about this before our meeting where we would talk about a solution. 

All other comments have been addressed and pushed. 

@joelostblom should I merge now?

**Comment by joelostblom** [comment]
@hfboyce Yes, just resolve all the comments that you have addressed and leave the one that are for me open

**Comment by hfboyce** [comment]
@joel I resolved everything in this PR including the comments on the slide so you may want to hop on this branch and take a look. I think this is ready to merge.

**Comment by hfboyce** [comment]
Can I merge this into master @joelostblom ?

**Comment by joelostblom** [comment]
Yup, thanks for resolving the comments!

**Pull request merged**